internal/testutil/acc/cluster.go

```diff
@@ -7,18 +7,29 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 )
 
-func GetClusterInfo(projectID string) (clusterName, clusterNameStr, clusterTerraformStr string) {
```

Comment:
knit: don't know if it's better to create a clusterInfo struct to be returned as we're getting more returned values

Reply:
also not in this PR but as it's getting bigger it may be nice to have some tests for it